x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/application/alert_type_registry.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { AlertTypeModel } from '../types';
+
+export class AlertTypeRegistry {
```

Comment:
It looks like ActionTypeRegistry and AlertTypeRegistry are identical - can we replace this with a single implementation using a Generic type for the Map?
Half the code to maintain :+1:

Reply:
done